.github/workflows/monitor-test-deploy.yml

```diff
@@ -0,0 +1,108 @@
+name: Teleport Monitoring
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - ./packages/teleport-monitoring
+  pull_request:
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./packages/teleport-monitoring
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache YARN dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            yarn-
+
+      - run: yarn --no-progress --non-interactive --frozen-lockfile
+
+      - run: yarn lint
+      - run: yarn format
+      - run: yarn test
+        env:
+          TEST_RPC: ${{ secrets.KOVAN_RPC_URL }}
+      - run: yarn typecheck
+
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    env:
+      REGION: us-east-1
+      SERVICE_NAME: teleport-monitor-prod
+      CLUSTER_NAME: teleport-monitor-prod
+
```

Comment:
@eskp You have to add here also `ECR_REPO_NAME`, since you're using it below in `docker build` and `docker push`